CompilerSource/ipc/Background.fbs

```diff
@@ -0,0 +1,22 @@
+include "Image.fbs";
+
+table Background {
+  name:string;
+  id:int;
+
+  transparent:bool;
+  smooth_edges:bool;
+  preload:bool;
+  use_as_tileset:bool;
+
+  tile_width:int;
+  tile_height:int;
+  horizontal_offset:int;
+  vertical_offset:int;
+  horizontal_spacing:int;
+  vertical_spacing:int;

```

Comment:
I could change this to the following now that Sprite has a Polygon subresource with a Point Type:
```flatbuffers
tile_size:Point;
offset:Point;
spacing:Point;
```
Should I do that? Would it make it clearer and more concise?